bin/sozo/src/commands/options/account.rs

```diff
@@ -40,21 +43,35 @@ impl AccountOptions {
     where
         P: Provider + Send + Sync,
     {
+        trace!(target: LOG_TARGET, account_options=?self, "Creating account.", );
         let account_address = self.account_address(env_metadata)?;
+        trace!(target: LOG_TARGET, ?account_address, "Account address determined.");
+
         let signer = self.signer.signer(env_metadata, false)?;
+        trace!(target: LOG_TARGET, ?signer, "Signer obtained.");
 
         let chain_id =
             provider.chain_id().await.with_context(|| "Failed to retrieve network chain id.")?;
+        trace!(target: LOG_TARGET, ?chain_id, "Chain ID obtained.");
+        let encoding = if self.legacy { 
+            trace!(target: LOG_TARGET, "Using legacy encoding.");
+            ExecutionEncoding::Legacy 
+        } else { 
+            trace!(target: LOG_TARGET, "Using new encoding.");
+            ExecutionEncoding::New 
+        };
 
-        let encoding = if self.legacy { ExecutionEncoding::Legacy } else { ExecutionEncoding::New };
-
+        trace!(target: LOG_TARGET, ?chain_id, ?encoding, "Creating SingleOwnerAccount.");
```

Comment:
```suggestion
        trace!(target: LOG_TARGET, ?encoding, "Creating SingleOwnerAccount.");
```
Chain id already printed above.